umm-spec-lib/src/cmr/umm_spec/validation/umm_spec_collection_validation.clj

```diff
@@ -24,6 +24,12 @@
       {field-path [(format "%%s minimum [%s] must be less than or equal to the maximum [%s]."
                            (str MinimumValue) (str MaximumValue))]})))
 
+(defn doi-format-warning-validation
+  "Validates that DOI is properly formatted."
+  [field-path value]
+  (when-not (re-matches #"\d\d\.\d\d\d\d(\.\d+)*\/.+" value)
+    {field-path [(format "DOI [%s] is improperly formatted." value)]}))
```

Comment:
Do we want to say what the proper format is?

Reply:
I doubt it, there is another ticket out there for removing regular expressions from error messages because they aren't user friendly enough.  Thats the reason I left it out.